security/bin/e2e.sh

```diff
@@ -50,8 +50,32 @@ if [[ "$HUB" =~ ^gcr\.io ]]; then
   gcloud docker --authorize-only
 fi
 
-# Build and push the Istio-CA docker image
-bin/push-docker.sh -h $HUB -t $TAG
+OUTPUT_DIR=`pwd`/docker
+
+# Generate certificate and private key from root
+echo 'Generate certificate and private key from root'
```

Comment:
'Generate certificate and private key for Istio CA'

Reply:
Done